external-test/.bazelrc

```diff
@@ -1 +1 @@
-build --incompatible_java_common_parameters=false
+build --cxxopt=-std=c++17 --incompatible_java_common_parameters=false --define=android_dexmerger_tool=d8_dexmerger --define=android_incremental_dexing_tool=d8_dexbuilder --nouse_workers_with_dexbuilder
```

Comment:
you might also want to add `--host_cxxopt=c++17`.  building @com_google_protobuf//:protoc requires this.